oss-internship-2020/pffft/README.md

```diff
@@ -1,14 +1,23 @@
 # Sandboxing PFFFT library
 
-Builder: CMake  
+Build System: CMake  
 OS: Linux
 
+### Check out the PFFFT library & CMake set up
+`git submodule add https://bitbucket.org/jpommier/pffft.git`
```

Comment:
This is actually what _you_ did to add the submodule to the project, not what your users should do after checkout.

If I want to use your code, I need to checkout SAPI (this repo) and then:
```bash
git submodule update --init --recursive
```
To actually get the code for pffft.